api/version.go

```diff
@@ -54,8 +54,8 @@ func VersionForType(nodeType NodeType) (Version, error) {
 
 // semver versions of the rpc api exposed
 var (
-	FullAPIVersion0 = newVer(1, 3, 0)
-	FullAPIVersion1 = newVer(2, 1, 0)
+	FullAPIVersion0 = newVer(1, 3, 1)
```

Comment:
@Stebalien @magik6k is there anything else needed to finish bumping the api version?

Reply:
I think that is it.